data_conventions.yaml

```diff
@@ -56,32 +56,56 @@ standard_tags:
         type: string
         comment: 'Either "client" or "server" for the appropriate roles in an RPC'
         examples: '"client", "server"'
- 
+
     'peer.hostname':
         type: string
         comment: 'Remote hostname'
         examples: '"opentracing.io", "internal.dns.name"'
- 
+
     'peer.ipv4':
         type: string
         comment: 'Remote IPv4 address as a .-separated tuple'
         examples: '"127.0.0.1"'
- 
+
     'peer.ipv6':
         type: string
         comment: 'Remote IPv6 address as a string of colon-separated 4-char hex tuples'
         examples: '"2001:0db8:85a3:0000:0000:8a2e:0370:7334"'
- 
+
     'peer.port':
         type: integer
         comment: 'Remote port'
         examples: '80'
- 
+
     'peer.service':
         type: string
         comment: 'Remote service name (for some unspecified definition of "service")'
         examples: '"elasticsearch", "a_custom_microservice", "memcache"'
- 
+
+    #####################################################################
+    # DATABASE tags
+    #####################################################################
+
+    'db.type':
+        type: string
+        comment: 'Database type'
+        examples: 'For any SQL database, "sql". For others, the lower-case database category, e.g. "cassandra", "hbase", or "redis".'
+
+    'db.instance':
+        type: string
+        comment: 'Database instance name'
+        examples: 'In java, if the jdbc.url="jdbc:mysql://127.0.0.1:3306/customers", instance name is "customers".'
```

Comment:
@yurishkuro , I think this again. Still think `db.instance` can be an optional tag.

Let's say, two types of people can instrument the db access from application side, 1) application developers, 2)ORM/DB component developers. For type(1), your point is true, but for type(2), they can get a `db.url`, and can get `db.instance` value easily. At same time, they can keep these info in their own Connection/Statement object(pooled object), when call `statement.execute`, this tag can be tagged.

So, IMO, `db.instance` works. Not be needed very time.